components/org.wso2.carbon.metrics.core/src/main/java/org/wso2/carbon/metrics/core/Gauge.java

```diff
@@ -16,7 +16,7 @@
 package org.wso2.carbon.metrics.core;
 
 /**
- * A metric to read a particular value
+ * A metric to read a particular value.
```

Comment:
Did you do this on purpose? Adding a full stop for every Java Doc comment? This is okay, but I just want to know. Many classes have changed just because of adding a full stop to Java Doc comments.